src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "../Introduce/Introduce.scss";
+
+const Introduce = () => {
+  return (
+    <div className="outLine">
```

Comment:
```suggestion
    <div className="introduce">
```
아까 말씀드린 것처럼 가장 부모 태그의 className은 컴포넌트의 이름과 동일하게 해주세요
단, 소문자로 시작해야 합니다!